src/main/scala/org/broadinstitute/dsde/firecloud/model/ESMapping.scala

```diff
@@ -1,7 +1,81 @@
 package org.broadinstitute.dsde.firecloud.model
 
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol.AttributeFormat
+import org.broadinstitute.dsde.firecloud.model.Attributable.AttributeMap
+import spray.json._
+import spray.json.DefaultJsonProtocol._
+import spray.json.{JsObject, JsValue}
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
+
 case class AttributeDefinition(properties: Map[String, AttributeDetail])
 case class AttributeDetail(`type`: String, items: Option[AttributeDetail]=None)
 
 case class ESDatasetProperty(properties: Map[String, ESDetail])
 case class ESDetail(`type`: String)
+
+
```

Comment:
Maybe rename this file since it's doing much more than elastic search mappings now